src/scripts/selectors.ts

```diff
@@ -58,13 +58,23 @@ const generateSelectors = async (): Promise<void> => {
   const rootSelectorCss = `@custom-selector :--root [${attr}='${value}'];`
   const rootSelectorJs = `case ":--root": return "[${attr}='${value}']"`
 
+  // Add an additional `[class*=language-]` qualified selector to Code nodes, to math PrismJS’s CSS specificity and
+  // allow styling tokenized elements.
+  const prismCompatSelector = (itemtype: string, schemaURL: string): string => {
+    const selector = `[itemtype~='${schemaURL}']`
+
+    return itemtype.toLowerCase().includes('code')
```

Comment:
`itemtype.toLowerCase().includes('code')` is too broad. This should only be needed for `CodeFragment` and `CodeBlock` (`Code ` is a base type, that is not instantiated, `CodeError`does not contain code etc).